.github/workflows/ci.yml

```diff
@@ -0,0 +1,27 @@
+name: App CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-node@v1
+      with:
+        node-version: 15.x
+    
+    - name: Install dependencies 
+      run: yarn install
+
+    - name: Format code
+      run: yarn run format
```

Comment:
I wouldn't run formatting during CI. We have a pre-commit git hook that will make sure files are formatted before being committed to the repo.

Reply:
updated and removed format code in CI